stl/inc/chrono

```diff
@@ -5232,6 +5233,7 @@ namespace chrono {
 
             const auto _Pos = _STD _Copy_unchecked(_Sv._Unchecked_begin(), _Sv._Unchecked_end(), _Specs._Fill);
             _STD fill(_Pos, _STD end(_Specs._Fill), _CharT{});
+            _Specs._Fill_length = static_cast<uint8_t>(_Sv.size());

```

Comment:
I observe that we could get rid of the zero-filling now that we store the length. (And in `<format>`.)

Reply:
hmm, I kept it because it makes things more consistent and it's not a huge cost, but we could get rid of it I suppose.